src/tbbmalloc/proxy.cpp

```diff
@@ -24,15 +24,17 @@
 // of aligned_alloc as required by new C++ standard, this makes it hard to
 // redefine aligned_alloc here. However, running on systems with new libc
 // version, it still needs it to be redefined, thus tricking system headers
-#if defined(__GLIBC_PREREQ) && !__GLIBC_PREREQ(2, 16) && _GLIBCXX_HAVE_ALIGNED_ALLOC
+#if defined(__GLIBC_PREREQ)
+#if !__GLIBC_PREREQ(2, 16) && _GLIBCXX_HAVE_ALIGNED_ALLOC
 // tell <cstdlib> that there is no aligned_alloc
 #undef _GLIBCXX_HAVE_ALIGNED_ALLOC
 // trick <stdlib.h> to define another symbol instead
 #define aligned_alloc __hidden_redefined_aligned_alloc
 // Fix the state and undefine the trick
 #include <cstdlib>
 #undef aligned_alloc
-#endif // defined(__GLIBC_PREREQ)&&!__GLIBC_PREREQ(2, 16)&&_GLIBCXX_HAVE_ALIGNED_ALLOC
+#endif // defined(__GLIBC_PREREQ)
+#endif // !__GLIBC_PREREQ(2, 16)&&_GLIBCXX_HAVE_ALIGNED_ALLOC
```

Comment:
Not a big deal because these are comments, but these lines should be swapped
```suggestion
#endif // !__GLIBC_PREREQ(2, 16)&&_GLIBCXX_HAVE_ALIGNED_ALLOC
#endif // defined(__GLIBC_PREREQ)
```

Reply:
Done

Reply:
Thanks!  It'd be good to target `master`, since `tbb_2019` isn't maintained anymore according to previous comments :wink:  You'll need to patch a different file though, it's called `src/tbbmalloc_proxy/proxy.cpp` now

Reply:
Do you think that's really worth it or will this new PR will be kept unmerged like this one?

Reply:
I don't have many insights since I'm not a maintainer, but I guess it won't ever be merged as it is since it's targeting a dead branch